src/clj_kondo/impl/linters.clj

```diff
@@ -191,6 +191,21 @@
                    (str ns-name "/" fn-name)
                    (some? const-true))))))))
 
+(defn- lint-missing-clause-in-try
+  [call]
+  (when (utils/one-of (:name call) [try try+])
+    (let [clauses #{'catch 'finally}
+          tokens (->> (get-in call [:expr :children])
```

Comment:
I'd prefer it if we could do the linting here:

https://github.com/borkdude/clj-kondo/blob/846bdb9a4f5f2b39bb6e96c6b3a7f3a4c9afcd0c/src/clj_kondo/impl/analyzer.clj#L824

We already walk the forms over there and have all the information that is needed to create the warnings. This avoids having to walk the AST twice.

Reply:
I moved the linter to `analyze-try` check it please.